README.md

```diff
@@ -496,57 +507,59 @@ These are some Store implementations authored and maintained by the community. T
   }
 ```
 
-# RFC6265bis
+## RFC 6265bis
 
-Support for RFC6265bis revision 02 is being developed. Since this is a bit of an omnibus revision to the RFC6252, support is broken up into the functional areas.
+Support for RFC 6265bis revision 02 is being developed. Since this is a bit of an omnibus revision to the RFC 6252, support is broken up into the functional areas.
```

Comment:
[RFC6265bis](https://datatracker.ietf.org/doc/draft-ietf-httpbis-rfc6265bis/) is currently at revision 09.  Should this be updated?

Reply:
I would think so, but that decision sounds like it should come from someone on the dev side of this discussion.

Reply:
We may want to drop the revision version piece here; we are still working to develop full support for RFC6265bis, I believe.  @ruoho

Reply:
Is support still under development, or has it been completed?